### DISCUSSION TITLE
Does there are some examples code?

### DISCUSSION THREAD

**Discussion by JiuRanYa** [clauderic/dnd-kit#1509] [Q&A]
Where I can get this example code: https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/docs/examples-pages-layout--grid

**Comment by ianpaschal** [comment]
Upvoted. I've been checking out this library today and the examples are sorely lacking. Storybook is awful for this sort of thing. I've gone digging in the stories folder, but of course any given example is spread across 5+ files with dozens and dozens of lines of props which are tough to figure out which ones are being passed through and are used.

Please, please add proper examples to the documentation.
